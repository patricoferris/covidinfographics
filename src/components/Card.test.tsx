import React from 'react'
import SimpleCard from './Card'
import renderer from 'react-test-renderer'

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <SimpleCard title="my title" content="some content">
      <h1>A child component</h1>
    </SimpleCard>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
