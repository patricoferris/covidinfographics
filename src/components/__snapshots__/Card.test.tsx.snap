// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div
  className="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="MuiCardContent-root"
  >
    <h2
      className="MuiTypography-root makeStyles-title-3 MuiTypography-h5 MuiTypography-gutterBottom"
    >
      my title
    </h2>
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      some content
    </p>
    <h1>
      A child component
    </h1>
  </div>
</div>
`;
